tasks.py

```diff
@@ -589,6 +589,8 @@ def check_schema(context, api_version=None):
         "verbose": "Enable verbose test output.",
         "append": "Append coverage data to .coverage, otherwise it starts clean each time.",
         "skip_docs_build": "Skip (re)build of documentation before running the test.",
+        "report": "Generate Performance Testing report in the terminal. Has to set GENERATE_PERFORMANCE_REPORT=True in settings.py",
+        "generate_report": "Generate a new performance testing report to report.json. Has to set GENERATE_PERFORMANCE_REPORT=True in settings.py",
```

Comment:
"Report" to me is more synonymous with coverage report than performance. And the report json would typically be referred to as snapshot.

So maybe something more like:
```suggestion
        "performance_report": "Generate Performance Testing report in the terminal. Has to set GENERATE_PERFORMANCE_REPORT=True in settings.py",
        "performance_snapshot": "Generate a new performance testing report to report.json. Has to set GENERATE_PERFORMANCE_REPORT=True in settings.py",
```